protocols/yungapesquad/overview.md

```diff
@@ -0,0 +1,22 @@
+## Yung Ape DAO
+
+Yung Ape DAO is the governing body of the Yung Ape Squad that works in harmony with the project team and the community. The main responsibilities of the DAO are to track and approve expenditures, decide on project activations, fund growth on multiple verticals.
+
+## Yung Ape DAO Treasury
+
+The Yung Ape DAO treasury will start with a base treasury of 25% of funds raised through minting. 5% of the royalties will flow through from secondary sales into the treasury continuously. The use of these funds will be a collective decision.
+
+The treasury will focus to cover the following expenses: Marketing, staking and tokenomics development, liquidity provision, game development, technical maintenance tasks. 
+
+The spending should focus on the growth of the project and support a long-term sustainable strategy for the well-being of the DAO members.
+
+## Yung Ape Proposals
+
+Yung Ape Proposals (YAPs) are proposals that are anticipated to play a big role in the future of the project. YAPs can be perceived as alignments that drive action points, the biggest difference for Young Ape DAO will be the fact that everybody will have a voice.
+
+The YAPs process and the project’s overall development are supported by a dedicated team who will be responsible for bringing these proposals to the DAO. DAO will get to choose whether to pursue the proposed strategy or not.
+
+### Additional Resources
```

Comment:
Missing `Join our DAO` section.
And I think that this should be a `#` level heading